docs/api/Auth0.ManagementApi.Clients.BaseClient.html

```diff
@@ -293,6 +294,49 @@ <h5 class="returns">Returns</h5>
       <tr>
         <td><span class="xref">System.Uri</span></td>
         <td><p>Completed absolute URI for the request.</p>
+</td>
+      </tr>
+    </tbody>
+  </table>
+  <a id="Auth0_ManagementApi_Clients_BaseClient_EncodePath_" data-uid="Auth0.ManagementApi.Clients.BaseClient.EncodePath*"></a>
+  <h4 id="Auth0_ManagementApi_Clients_BaseClient_EncodePath_System_String_" data-uid="Auth0.ManagementApi.Clients.BaseClient.EncodePath(System.String)">EncodePath(String)</h4>
+  <div class="markdown level1 summary"><p>Encode a value so it can be successfully used in the path.</p>
+</div>
+  <div class="markdown level1 conceptual"></div>
+  <h5 class="decalaration">Declaration</h5>
+  <div class="codewrapper">
+    <pre><code class="lang-csharp hljs">protected string EncodePath(string value)</code></pre>
+  </div>
+  <h5 class="parameters">Parameters</h5>
+  <table class="table table-bordered table-striped table-condensed">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Name</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><span class="xref">System.String</span></td>
+        <td><span class="parametername">value</span></td>
+        <td><p>Value to encode for the path.</p>
+</td>
+      </tr>
+    </tbody>
+  </table>
+  <h5 class="returns">Returns</h5>
+  <table class="table table-bordered table-striped table-condensed">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><span class="xref">System.String</span></td>
+        <td><p>URI encoded/escaped value that can be used in the path.</p>
```

Comment:
Docs not run when implemented a while ago https://github.com/auth0/auth0.net/commit/30c38b935b61e879ba2c09d0652b6e063fd63ef6